clientd3d/ownerdrw.c

```diff
@@ -341,16 +380,39 @@ void DrawOwnerListItem(const DRAWITEMSTRUCT *lpdis, Bool selected, Bool combo)
    SetBkMode(lpdis->hDC, OPAQUE);
    obj = (object_node*)lpdis->itemData;
 
-   hColorBg = GetBrush(GetItemListColor(lpdis->hwndItem, (selected? SEL_BGD : UNSEL_BGD)));
+   hColorBg = GetBrush(
+                  GetItemListColor(
+                     lpdis->hwndItem,
+                     (selected? SEL_BGD : UNSEL_BGD),
+                     item_rarity_value));
+
    if ((style & OD_ONLYSEL) && (style & (OD_DRAWOBJ | OD_DRAWICON)))
-      hColorBg = GetBrush(GetItemListColor(lpdis->hwndItem, UNSEL_BGD));
+      hColorBg = GetBrush(
+                     GetItemListColor(
+                        lpdis->hwndItem,
+                        UNSEL_BGD,
+                        item_rarity_value));
```

Comment:
here too

Reply:
Changed parameter to `ITEM_RARITY_GRADE_NORMAL` constant here also, as requested.